Assignment4/Assignment4.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+
+function readAppendWrite (readFile, appendFile, writeFile) {
```

Comment:
`readAppendWrite (readFile, appendFile, writeFile)` should be `readAppendWrite(readFile, appendFile, writeFile)`.
Problem Type: Spacing